docs/content/iota-evm/how-tos/core-contracts/objects/introduction.mdx

```diff
@@ -0,0 +1,18 @@
+---
+description: How to handle native Objects on L2 and use them as ERC721
+image: /img/logo/WASP_logo_dark.png
+tags:
+  - nft
+  - evm
+  - how-to
+teams:
+  - iotaledger/l2-smart-contract
+---
+import ERC721 from '../../../../_snippets/iota-evm/how-tos/ERC721.md';
+
+# Coins and Objects
+
+<ERC721/>
+
+The IOTA L1 can create Objects, also called native Objects. To use these Objects on L2, you have
+an ERC20 contract called `ERC721Objects`, which contains all L1 Objects owned by the chain. The following guides will show you how to [mint your own L1 Object from L2](./mint-Object.mdx) and use it with the `ERC721Objects` contract.
```

Comment:
This section doesn't really make sense.

We can remove the ERC721 component. And also there is no such thing as ERC721Objects afaik.

So I would just say something like this maybe:
```suggestion
The IOTA L1 is Object based. The following guides will show you how to interact with this Objects from L2.
```